azure-iot-device/tests/iothub/pipeline_test.py

```diff
@@ -0,0 +1,20 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
```

Comment:
Same comments as in common,

1) this should be pipeline_test_helpers, if it has to exist at all (consider maybe putting it in the conftest.py?)
2) this and all other pipeline specific config/fixtures/etc. should be scoped to only apply to places where we are testing those things